### PR TITLE
multiple creation already selected users remain saved

### LINKED ISSUE #1371: 🔧 [Refactor] ADMINAREA - Multiple creation  already selected users remain saved
## 🔧 Refactor ticket
Multiple creation: I had selected 2 users and had to go back to the user search for the third user to see if his account was confirmed. When I then went back to multiple creation, the 2 users already selected were deleted. I practically had to start all over again.
Task: I want to be able to go out of the multiple creation and back again and keep my already selected users remain saved.

Mehrfachschöpfung: ich hatte 2 Nutzer ausgewählt und musste für den dritten Nutzer zurück zur Nutzer-suche, um nachzusehen, ob sein Konto bestätigt ist. Als ich dann wieder in die Mehrfachschöpfung zurückgegangen bin, waren die beiden bereits ausgewählten 2 Nutzer gelöscht. Ich musste praktisch von vorne anfangen.
Aufgabe: Ich möchte aus der Mehrfachschöpfung herausgehen können und wieder zurück und meine bereits gewählten Nutzer bleiben gespeichert.

unknown

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
cache for selected users in the multiple creation. if i have pre-saved users for a multiple creation and then have to leave the page and call it up again. the pre-saved users are entered in the table again and removed from the search table.

### Issues
- fixes #1371 
- merge #1375 

ezgif com-gif-maker

### CODE REVIEW

**Comment by Mogge** [comment]
@ogerly Ich habe mal im größeren Stil refactored.

**Comment by Mogge** [comment]
Was noch fehlt, sind die IDs der ausgewählten User an den Server zu übermitteln, dass diese nicht neu ausgeliefert werden. Wenn man viele User ausgewählt hat, ist die Liste links fast leer.

File: admin/src/pages/Creation.spec.js [hunk 1]
```diff
@@ -93,121 +108,141 @@ describe('Creation', () => {
...
+          .at(1)
+          .vm.$emit('remove-item', {
+            userId: 2,
+            firstName: 'Benjamin',
+            lastName: 'Blümchen',
+            email: 'benjamin@bluemchen.de',
+            creation: [800, 600, 400],
+            showDetails: false,
           })
+      })
```
Comment (ulfgebhardt) [inline admin/src/pages/Creation.spec.js:184]: `remove item` using `push-item` event - have I missunderstood something?

i think I do not really understand the test?! Is this the other list(source list) from which the emits stem from?

**Review by ulfgebhardt — APPROVED** [review]
I did not understand this one test, but other than that it looks very good - also tried it locally - works like a charm

**Pull request merged**